.github/workflows/rust.yml

```diff
@@ -102,6 +110,7 @@ jobs:
           exit 1
         fi
 
+
```

Comment:
Minor: not sure if this new empty line was intended?

Reply:
Thanks :)